blockchain/util/crypto.py

```diff
@@ -155,6 +156,8 @@ def sign_verification_record(signatory,
             - arbitrary string
     :return: verification record with assembled signature field
     """
+    # signature, transaction_hash = \
```

Comment:
I think that might've been something to do with subscriptions?... I don't recognize it as anything that goes with timestamping. Removed for now but can be added back later if needed.